pkgs/development/tools/vulkan-validation-layers/default.nix

```diff
@@ -18,6 +19,17 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [ "-DGLSLANG_INSTALL_DIR=${glslang}" ];
 
+  # Help vulkan-loader find the validation layers
+  setupHook = writeText "setup-hook" ''
+    export VK_LAYER_PATH=@out@/share/vulkan/explicit_layer.d
```

Comment:
A more graceful approach here would be to append `@out@/share` to `XDG_DATA_DIRS`, therefore not clobbering the visibility of any other layers in the environment.

Reply:
That is a great suggestion! I'll check to see if it works.

Reply:
It works perfectly, thanks again.